src/v/raft/tests/replication_monitor_tests.cc

```diff
@@ -91,20 +91,31 @@ TEST_P_CORO(monitor_test_fixture, truncation_detection) {
 
     for (auto& [id, node] : nodes()) {
         if (id == leader) {
-            node->on_dispatch(
-              [](model::node_id, raft::msg_type) { return ss::sleep(3s); });
+            node->on_dispatch([](model::node_id, raft::msg_type mt) {
```

Comment:
should this diff be in the `dev` too? This is not specific to 24.3.x, right?

Reply:
yes, will prepare correct fixes